jenkins/internal-ci/centos-7.8.2003/main/cortx-ha.groovy

```diff
@@ -218,11 +218,11 @@ def getAuthor(issue) {
     def author= ""
     def response = ""
     // Grab build information
-    for (int i = 0; i < changeLogSets.size(); i++){
+    for (int i = 0; i < changeLogSets.size(); i++) {
```

Comment:
Please test one job with this change to ensure nothing breaks.

Reply:
According to codacy this the best practice.

Reply:
Yes, but the code should also with Codacy's recommended changes. We earlier observed that sometime Codacy recommendations caused script failures.

Reply:
We can test but executing each script will take a lot of time. Some scripts are dependent on some other scripts as well.